### DISCUSSION TITLE
Python wrapper

### DISCUSSION THREAD

**Discussion by harrylojames** [observablehq/plot#1554] [General]
Hi, 

First off big fan of this project! 

Curious, are there any plans to officially support a Python wrapper? I am aware of this [project ](https://github.com/juba/pyobsplot) and have played with mixing Observable and Python with Quarto. 

Many thanks,

Harry

**Comment by mbostock** [comment]
Thank you. No, we have no plans to support a Python wrapper; we are focused on JavaScript. It’s nice that Julien Barnier has made one though!

**Reply by harrylojames** [reply]
Appreciate the reply! Not to worry :)
